sky/server/requests/executor.py

```diff
@@ -448,6 +453,31 @@ def start(deploy: bool) -> List[multiprocessing.Process]:
     return sub_procs
 
 
+def _slow_start_long_workers(long_workers: List[multiprocessing.Process],
+                             delay: float = 2.0) -> None:
+    """Start long workers with TCP-like slow start.
+
+    This is to avoid overwhelming the CPU on startup.
+
+    Args:
+        long_workers: The list of long worker processes to start.
+        delay: The delay between starting each worker process, default to 2.0
+            seconds based on profile.
+    """
+    workers_left = len(long_workers)
+    batch_size = 1
+    while workers_left > 0:
+        current_batch = min(batch_size, workers_left)
+        for i in range(current_batch):
+            worker_idx = len(long_workers) - workers_left + i
+            long_workers[worker_idx].start()
+        workers_left -= current_batch
+        if workers_left <= 0:
+            break
+        time.sleep(delay)
```

Comment:
Just wondering if the delay is a bit too large?

Reply:
The delay is conservative intentionally to ensure that the processes in previous batch have completed the CPU-intensive module loading phase when we launch the next batch. So that we don't have to bother probing the process status and CPU usage.

Reply:
Profiling results were added to the PR description